### DISCUSSION TITLE
Looks like there's a few us making our own...

### DISCUSSION THREAD

**Discussion by wamcvey** [stevenbennett/DIYson#34] [General]
Binged through the youtube series yesterday and now am looking at building my first (probably won't be my last) lamp. So I was checking out the bill of materials and noticed that we seem to be having an [influence on Amazon's buying trends](https://i.imgur.com/A9wlOjA.png) for these products. Seems there's at least a few us making our own!

**Comment by stevenbennett** [comment]
I want to make a community site where folks can share photos and details of their builds (both DIYson and DIYson Express) (roughly comparable to what Zack Freedman's community has done with the [Gridfinity Catalog](https://gridfinity.xyz/). Does anyone know of a web platform that would support something like this?

**Reply by wamcvey** [reply]
Buddyboss under Wordpress could probably handle that. Alternatively, /r/DIYson is still available on Reddit or you could start a discord instance and set up a channel for works in progress. Buddyboss (and wordpress in general) is a bit of a pain in the ass to administer, so if it were me, I'd probably lean heavily towards building your community on Reddit or Discord.

**Comment by legopc** [comment]
![photo1677879341](https://user-images.githubusercontent.com/121928053/222833895-4148feb9-22ea-4a76-bcb9-b5012511f31b.jpeg)
How about a thread with some build pics. Here is mine :) Please excuse the mess.

**Comment by SK00o** [comment]
Work is progress here, (picture is from some weeks ago) at the moment I have a working power slide solution with nickel stips and magnets no capacitor needed
![20230323_205812](https://user-images.githubusercontent.com/132890329/236782614-c36c61dd-86a1-4cf6-9616-df9ce775158d.jpg)

**Reply by SK00o** [reply]
I'm using TTP223B (red variant)  for the three touch buttons on top,  MEAN WELL  LDD-700LW in combination with CMA1303-0000-000C0U0A40G led

**Reply by clemensul** [reply]
Hi - what is your base made of? Looks good!

**Reply by SK00o** [reply]
It's a base from a old desk lamp. Because it's heavy it also doubles as a counterweight for the lamp.
